back/social/templates/social/club_edit.html

```diff
@@ -90,52 +131,52 @@
                                         {% if membership.is_admin %}
                                             <span>(admin)</span>
                                         {% endif %}
+                                    </div>
+                                </div>
+                                <div class="centered-div">
+                                    <form action="" method="post" enctype="multipart/form-data">
+                                        {% csrf_token %}
+                                        <input type="hidden" value="{{membership.student.id}}" name="student_id">
+                                        <input class="button blue-button" type="submit" name="Vieux" value="Inactif">
+                                        <input class="button red-button" type="submit" name="Supprimer" value="Supprimer">
+                                    </form>
+                                </div>
+                                <br><br>
+                            </div>
+                        {% endfor %}
+                    </div>
+                {% endif %}
+                {% if all_active_club_memberships %}
```

Comment:
There is a problem in the logic here for the edge case where there is no longer any active member the buttons wont show. A variable all_inactive_club_membership  would be better